pkgs/development/libraries/libnbd/default.nix

```diff
@@ -8,6 +8,8 @@
 , fuse
 , fuse3
 , gnutls
+, ubdsrv
```

Comment:
Are you interested in maintaining libnbd too?

Reply:
Yes sure, I'll add myself